cmd/test/executorRun.go

```diff
@@ -42,12 +43,22 @@ func newArwenTestExecutor() *arwenTestExecutor {
 	}
 }
 
+func (te *arwenTestExecutor) setErc20Path(erc20Path string) *arwenTestExecutor {
+	te.erc20Path = erc20Path
+	return te
+}
+
 // ProcessCode takes the contract file path, assembles it and yields the bytecode.
 func (te *arwenTestExecutor) ProcessCode(testPath string, value string) (string, error) {
 	if len(value) == 0 {
 		return "", nil
 	}
-	fullPath := filepath.Join(testPath, value)
+	var fullPath string
+	if value == "wrc20_arwen.wasm" && te.erc20Path != "" {
+		fullPath = te.erc20Path
+	} else {
+		fullPath = filepath.Join(testPath, value)
+	}
```

Comment:
Why is `value == "wrc20_arwen.wasm"` necessary here? Can't it be treated like any other WASM binary?

Reply:
we have the same tests for 3 different binaries, it was a quick and dirty solution. I'll replace it with something more palatable